lighthouse-core/gather/gatherers/seo/tap-targets.js

```diff
@@ -298,10 +294,10 @@ class TapTargets extends Gatherer {
       ${getClientRects.toString()};
       ${hasTextNodeSiblingsFormingTextBlock.toString()};
       ${elementIsInTextBlock.toString()};
-      ${getRectArea.toString()};
-      ${getLargestRect.toString()};
-      ${getRectCenterPoint.toString()};
-      ${rectContains.toString()};
+      ${RectHelpers.getRectArea.toString()};
```

Comment:
Nit: Can we remove these unused RectHelpers functions? Looks like only getRectCenterPoint() is used so the others aren't needed.

Reply:
> > Just seems important to increase test coverage of our bundle as we increasingly do more things to it
> 
> This seems _really_ important, since it's not going to be obvious that writing normal JS might break when minified (same for the fix needed in [d657e76](https://github.com/GoogleChrome/lighthouse/commit/d657e76273f40f41cbe15720f55e5ce0c931a54e)).

I'll open a PR now that runs every smoke test on the bundle.

> Since we're going all in on minification, it does seem like turning on all smoke tests for the bundle is necessary now.
> 
> > The tap-targets gatherer was always failing in CDT
> 
> But not caught in the devtools smoke tests, right?

Sorry, that was very vague, I updated the first comment.

Reply:
nice catch